stix_shifter/stix_transmission/src/modules/bigfix/bigfix_status_connector.py

```diff
@@ -57,8 +57,6 @@ def update_query_status(self, return_obj, response_dict, client_count):
         return_obj = self._get_progress_status(client_count, reporting_agents, return_obj)
         if client_count <= reporting_agents:
             return_obj['status'] = Status.COMPLETED.value
-            if total_results <= 0:
-                return_obj['status'] = Status.ERROR.value
         else:
```

Comment:
We are having some issue in pipeline testing while the progress is less than 100 but the status is COMPLETED. We are getting some conflicting status. I know we have discussed about this a lot earlier. Can put a check before returning the status to set the progress value to 100 when the status is COMPLETED? we can still keep the progress threshold check the same.